### ISSUE TITLE
Speaker and microphone dead in secure calls

### ISSUE THREAD

**Issue by AlfonsoMuskedunder** [WhisperSystems/Signal-Android#4267]
This morning I tried to establish a couple of secure calls between two Android Signal phones. Only the first time everything worked fine; however, thereafter
* everytime, after having successfully established a connection, my phone would not transmit anything I said nor would I hear anything my friend said. It's as if my phone was dead, even though the app indicates are normally functioning call. I adjusted my phone's volume during such calls, it doesn't seem like the volume had been set to zero or something.
* A couple of times the connection would sort of disappear with both phones stating in the top bar that a phone call to an unknown recipient was active (clicking on that notification gave a screen identical to the one depicted in https://github.com/WhisperSystems/TextSecure/issues/4234.)
* In the most recent attemts to call my friend her phone wouldn't even ring anymore

In case someone found the time to have a look, here are the logs. My phone (Signal 3.0.3, Android 4.2.2): https://gist.github.com/4ed595aa929def90900a and (most recently with the failed call initialization) https://gist.github.com/a2ac6019b5f7ec93c826
Other phone (Signal 3.0.3, Android 4.1.2): https://gist.github.com/7bc9ccc7422fa9709896

**Comment by AlfonsoMuskedunder** [comment]
After reinstalling Signal I just established a normally functioning secure call with the same friend. (Reinstalling also solved https://github.com/WhisperSystems/TextSecure/issues/4100. At some point the interaction between the phone and TS must have been messed up. Perhaps an OS update; I don't remember the timing of the related incidents precisely.) I'll reopen if the problem reappears.

**Comment by AlfonsoMuskedunder** [comment]
OK, closing this was premature. I just tried to establish three subsequent calls with a different contact than the one mentioned above. Two times or so I couldn't hear anything (not even very silent, as described in another issue); once I heard my friend for a tiny fraction of a second.
https://gist.github.com/1d9a6fa6722eb2db4260
(Yesterday we succeeded in making test calls immediately after my friend had upgraded to Signal.)

**Comment by moxie0** [comment]
My hope is that 0e291a9509bb364b7595b88dae452df7c714ed7a fixes this.

**Comment by AlfonsoMuskedunder** [comment]
Apparently not. My friend mentioned in my first post and I tried to establish a couple of calls this morning. The first time a conncection was established (at least the ping sound rang), but again neither of us could hear a thing. In the attempts thereafter, attempts to call ended in this screen, without the other phone ringing:
![screenshot_2015-11-06-09-57-39](https://cloud.githubusercontent.com/assets/10466043/10993536/d00fa7e8-846f-11e5-99ee-810cb0273f0d.png)
This happened when either of us tried to call the other one.

My phone's log: https://gist.github.com/af186a32ee499a28a098
The other phone's log: https://gist.github.com/1c8848e44b00366e1554

**Comment by AlfonsoMuskedunder** [comment]
Oh boy. Today I successfully established three or so subsequent calls with my friend. They all got interrupted suddenly without notification after about two minutes. I would count that as small progress compared with the state described above.

After that however, when I tried to initiate a call, the screen would indicate (normally) the attempt, but no sound at all would ring. After closing and reopening the app, at least the sound that indicates the attempt to connect (not the free-line signal (?)) would ring. However, it's now impossible to call her.

Here's a log, in case they help: https://gist.github.com/d0a2dd1e2535c211a9c1
